lib/quark/compose.ex

```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```

Comment:
I still think this one seems unclear, maybe it should instead be:
```
Function composition, pipes a value to the first argument then that output is piped to the second argument.
```
Or something like that?